.vscode/launch.json

```diff
@@ -1,83 +0,0 @@
-{
```

Comment:
Not sure if .vscode should be in the commit...

Reply:
It shouldn't, somehow I didn't include it in the .gitignore